docs/source/en/task_summary.mdx

```diff
@@ -10,1125 +10,222 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
->>> not_paraphrase_results = tf.nn.softmax(not_paraphrase_classification_logits, axis=1).numpy()[0]
-
->>> # Should be paraphrase
->>> for i in range(len(classes)):
-...     print(f"{classes[i]}: {int(round(paraphrase_results[i] * 100))}%")
-not paraphrase: 10%
-is paraphrase: 90%
-
->>> # Should not be paraphrase
->>> for i in range(len(classes)):
-...     print(f"{classes[i]}: {int(round(not_paraphrase_results[i] * 100))}%")
-not paraphrase: 94%
-is paraphrase: 6%
-```
-</tf>
-</frameworkcontent>
-
-## Extractive Question Answering
-
-Extractive Question Answering is the task of extracting an answer from a text given a question. An example of a
-question answering dataset is the SQuAD dataset, which is entirely based on that task. If you would like to fine-tune a
-model on a SQuAD task, you may leverage the [run_qa.py](https://github.com/huggingface/transformers/tree/main/examples/pytorch/question-answering/run_qa.py) and
-[run_tf_squad.py](https://github.com/huggingface/transformers/tree/main/examples/tensorflow/question-answering/run_tf_squad.py)
-scripts.
-
-
-Here is an example of using pipelines to do question answering: extracting an answer from a text given a question. It
-leverages a fine-tuned model on SQuAD.
+But one of the key challenges Transformer architectures have helped is in low-resource languages. By pretraining on large amounts of speech data, finetuning the model on only one hour of labeled speech data in a low-resource language can still produce high-quality results compared to previous ASR systems trained on 100x more labeled data.
 
 ```py
 >>> from transformers import pipeline
 
->>> question_answerer = pipeline("question-answering")
-
->>> context = r"""
-... Extractive Question Answering is the task of extracting an answer from a text given a question. An example of a
-... question answering dataset is the SQuAD dataset, which is entirely based on that task. If you would like to fine-tune
-... a model on a SQuAD task, you may leverage the examples/pytorch/question-answering/run_squad.py script.
-... """
-```
-
-This returns an answer extracted from the text, a confidence score, alongside "start" and "end" values, which are the
-positions of the extracted answer in the text.
-
-```py
->>> result = question_answerer(question="What is extractive question answering?", context=context)
->>> print(
-...     f"Answer: '{result['answer']}', score: {round(result['score'], 4)}, start: {result['start']}, end: {result['end']}"
-... )
-Answer: 'the task of extracting an answer from a text given a question', score: 0.6177, start: 34, end: 95
-
->>> result = question_answerer(question="What is a good example of a question answering dataset?", context=context)
->>> print(
-...     f"Answer: '{result['answer']}', score: {round(result['score'], 4)}, start: {result['start']}, end: {result['end']}"
-... )
-Answer: 'SQuAD dataset', score: 0.5152, start: 147, end: 160
+>>> transcriber = pipeline(task="automatic-speech-recognition")
```

Comment:
Good question! I don't think there is an argument in the pipeline for setting a different language. You would need to use a multilingual model or one finetuned on a specific language. At the moment, I can't remember what the default model is for ASR. 🫣

We briefly address your question [here](https://huggingface.co/docs/transformers/main/en/quicktour#use-another-model-and-tokenizer-in-the-pipeline), but we don't explicitly say it. Maybe it can be clarified there or in the [pipeline tutorial](https://huggingface.co/docs/transformers/main/en/pipeline_tutorial)?